h/routes.py

```diff
@@ -58,6 +58,10 @@ def includeme(config):  # noqa: PLR0915
         "admin.email.preview.mention_notification",
         "/admin/email/preview/mention-notification",
     )
+    config.add_route(
+        "admin.email.preview.moderated_annotation_notification",
+        "/admin/email/preview/moderated-annotation-notification",
```

Comment:
Also for consistency with the other routes.

```suggestion
        "admin.email.preview.annotation_moderation_notification",
        "/admin/email/preview/annotation-moderated-notification",
```